plugins/monolith/ortac-monolith.opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+synopsis: "Monolith plugin got Ortac"
+description: "Monolith plugin got Ortac"
```

Comment:
Drive-by-comment: `for Ortac` (in both lines 2 and 3)

Reply:
Fixed. Thanks :-)